docs/content/concepts/io-management/io-managers.mdx

```diff
@@ -232,6 +269,28 @@ def my_job():
     op_2(op_1())
 ```
 
+### Per-input IO manager
+
+Just as with the inputs to assets, the inputs to ops can be loaded using custom logic. To set an IO manager for a particular input, use the `input_manager_key` argument on <PyObject module="dagster" object="In" />.
```

Comment:
I think it's worth clarifying that this is only for when you want to override the IO manager from the upstream output.